<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">* {
  border: none;
  background-color: transparent;
  color: #FFF; /* Specify a default color */
}

#centralwidget {
  background-color: white
;
}

#side_menu {
  background-color: #071e26;
  border-radius: 20px;
}

QPushButton {
  padding: 10px;
  background-color: #071e26;
  border-radius: 5px;
}

#main_body {
  background-color: #071e26;
  border-radius: 10px;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="header">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Menu</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/icons_folder/align-justify.svg</normaloff>:/icons/icons_folder/align-justify.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: black;</string>
            </property>
            <property name="text">
             <string>My app</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="side_menu" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="frame_4">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Item 1</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/New folder/airplay.svg</normaloff>:/icons/New folder/airplay.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>Item 2</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/New folder/award.svg</normaloff>:/icons/New folder/award.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>Item 3</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/New folder/coffee.svg</normaloff>:/icons/New folder/coffee.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>Item 4</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/New folder/music.svg</normaloff>:/icons/New folder/music.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_6">
               <property name="text">
                <string>Item 5</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/New folder/wifi.svg</normaloff>:/icons/New folder/wifi.svg</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="main_body">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
color:white;
font-family: &quot;Times New Roman&quot;, Times, serif;

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>22</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:25px;
</string>
              </property>
              <property name="text">
               <string>S.N</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>22</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:25px;
</string>
              </property>
              <property name="text">
               <string>Subject</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>22</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:25px;</string>
              </property>
              <property name="text">
               <string>    Upload Note</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>22</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:25px;</string>
              </property>
              <property name="text">
               <string>       View Note</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_6">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>Physics</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="upload_physics">
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
 border-bottom: 1px solid white;
border-radius:none;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>    Upload Note</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="view_physics">
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
 border-bottom: 1px solid white;
border-radius:none;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>View Notes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_20">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_21">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>Chemistry</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="upload_chemistry">
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
 border-bottom: 1px solid white;
border-radius:none;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>    Upload Note</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="view_chemistry">
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
 border-bottom: 1px solid white;
border-radius:none;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>View Notes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_22">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_23">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:30px;
font-family: &quot;Times New Roman&quot;, Times, serif;</string>
              </property>
              <property name="text">
               <string>Maths</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="upload_maths">
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
 border-bottom: 1px solid white;
border-radius:none;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>    Upload Note</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="view_maths">
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
 border-bottom: 1px solid white;
border-radius:none;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>View Notes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_24">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_25">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>Java</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="upload_java">
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
 border-bottom: 1px solid white;
border-radius:none;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>    Upload Note</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="view_java">
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
 border-bottom: 1px solid white;
border-radius:none;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>View Notes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_26">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_27">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>Python</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="upload_python">
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
 border-bottom: 1px solid white;
border-radius:none;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>    Upload Note</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="view_python">
              <property name="styleSheet">
               <string notr="true">background-color: Transparent;
color: white;
 border-bottom: 1px solid white;
border-radius:none;
font-family: &quot;Times New Roman&quot;, Times, serif;
font-size:30px;</string>
              </property>
              <property name="text">
               <string>View Notes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>main_body</zorder>
      <zorder>side_menu</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
